sass/_b3-support-reviews.scss

```diff
@@ -0,0 +1,120 @@
+.b3-support-reviews{
+  background: $background-B;
+  color: $white;
+  padding-bottom: 1em;
+  padding-top: 1em;
+}
+.reviews-container{
+  margin: auto;
+}
+.b3-support-reviews {
+ h1 {
+  font-family: "Raleway";
```

Comment:
This font-family is getting defined a lot -- this is one of those things where you want to have it applied more globally (rather than focusing in on the modular element). The same goes for the `font-weight` and the `font-size` -- in this case, it'd be better to have more global typographic rules that manage these settings, while the module itself manages the overall layout and structure of the module.